modules/moa-installing-prerequisites.adoc

```diff
@@ -24,7 +24,12 @@ This user will have Programmatic access enabled and the AdministratorAccess poli
 ====
 +
 . Install and configure the AWS CLI.
-.. Follow the AWS documentation to install the link:https://aws.amazon.com/cli/[aws-cli] for your operating system.
+.. Follow the AWS documentation to install and configure the link:https://aws.amazon.com/cli/[aws-cli] for your operating system.
+.. Optional: Set a default AWS region. `moactl` evaluates regions in the following priority order:
++
+... The region specified when running a `moactl` command with the `--region` flag.
+... The region set in the `AWS_DEFAULT_REGION` environment variable. See the link:https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html[AWS documentation for more details].
```

Comment:
It's better to include the title of the document that you're linking to instead of a generic reference, like:

```
See link:https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html[title] in the AWS documentation.
```

Reply:
But this is a much better way to present the information! Thank you. :)

Reply:
@kalexand-rh Hi, Kathryn. I updated the doc with Kevin's changes and he updated the manual preview link. So this is ready for a second round of peer review when you have a chance. Thank you!